spec/models/blog_post_spec.rb

```diff
@@ -135,4 +152,14 @@
     end
   end
 
+  describe "#set_slug" do
+    let(:title) { "blog post title" }
+    let(:blog_post) { create(:blog_post, :title => title) }
+
+    it "creates the slug from the title" do
+      expected_slug = title.parameterize
```

Comment:
I'm not sure this is testing the right thing. It relies on `parameterize` to do what it wants to do. So if you change what parameterize does, then the test still passes even if it shouldn't. This should be an integration test, not a unit test.